site/CLAUDE.md

```
@@ -10,6 +10,14 @@ npm run build    # Build to dist/
 npm run preview  # Preview production build
 ```
 
+**Before running Playwright tests** (`npm test`), build the WebContainer snapshot first:
+```bash
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)